src/data/navigation/sections/mesh.js

```diff
@@ -139,7 +139,31 @@ module.exports = [
             },
             {
                 title: "Extend your mesh",
```

Comment:
Upon review, I'm going to suggest an alternative page hierarchy that might make more sense. Technically `Custom Resolvers` **_IS_** the advanced feature. Let me know what you think.

```
Advanced Features
    > Custom Resolvers
        > Extend your Mesh
        > Programmatic Resolvers
        > Declarative Resolvers
        > Batching
```

Reply:
@amolina-adobe i think leaving `extend your mesh` at a higher level is more beneficial, because its meaning is a bit more accessible and it leaves more space for future non resolver based extensions.

- Extend your mesh
  - Extend your schema with AdditionalTypeDefs
  - Customize resolved values with AdditionalResolvers
    - Declarative resolvers
    - Programmatic resolvers
  - Batching

@revanth0212 what are your thoughts on this?

Reply:
Agreed to keep this organization 👍